contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -68,6 +68,11 @@
             data-test="duplicate-selected-btn"
             @click="duplicateNodes(selected)"
           />
+          <IconButton
+            icon="copy"
+            :text="$tr('')"
```

Comment:
You may add a string, `'Sort alphabetically'`, to to the `$trs` in this component. Then you may reference that string here.